test/godeps/lxd-agent.list

```diff
@@ -328,6 +330,7 @@ gopkg.in/mgo.v2/internal/json
 gopkg.in/tomb.v2
 gopkg.in/yaml.v2
 gopkg.in/yaml.v3
+k8s.io/utils/inotify
```

Comment:
this is incorrect, there is no reason for lxd-agent to need the inotify dependency, so it we should ensure that the ubuntupro package is split in such a way to achieve this.

We added this godeps check after the openfga functionality was added to catch just this sort of thing where unnecessary deps were being imported into lxd-agent rather than ensuring the separation of server and client imports.